HIF.md

```diff
@@ -0,0 +1,810 @@
+    this block in the command also disables parent following. The next
+    parameters set up the initial timings.
+
+ - `uint64_t bt_timestamp_us`  
+    Timestamp associated with the last received BT-IE from this node. The host
+    will use the `timestamp_us` value from [`CNF_DATA_TX`][tx-cnf] and
+    [`IND_DATA_RX`][rx].
+
+ - `uint16_t slot`  
+    Broadcast slot (from BT-IE).
+
+ - `uint32_t interval_offset_ms`  
+    Broadcast interval offset (from BT-IE).
+
+ - `uint32_t frame_counters[4]`  
+    Initial frame counters for this node for all keys (indices 1 through 4). As
+    described in ["Security"][sec], the RCP generally does not check for frame
+    counters since it is the responsability of the host. However this makes it
+    easy to break the RCP broadcast schedule by replaying a frame containing a
+    BT-IE. Thus the RCP can track the frame counters for a single parent. These
+    counters are not reset when installing a new key, so it is recommended to
+    always call [`SET_FHSS_FFN_BC`][bc] with updated counters right after
+    calling [`SEC_SET_KEY`][key].
+
+### `0x32 SET_FHSS_LFN_BC`
+
+Configure broadcast schedule for transmission to LFN.
+
+ - `uint16_t interval`  
+    LFN broadcast interval in milliseconds (from LBS-IE).
+
+ - `uint16_t bsi`  
+    LFN broadcast Schedule ID (from LBS-IE).
+
+ - `struct chan_seq`  
+   See ["Channel Sequence"][chan-seq].
+
+> [!WARNING]
+> The current RCP implementation uses the same set of parameters for FFN and
+> LFN brodcast channel sequences. Thus, [`SET_FHSS_FFN_BC`][bc] and
+> [`SET_FHSS_LFN_BC`][bc-lfn] override each other's channel sequence. The other
+> parameters are distinct.
+
+> [!NOTE]
+> The host is responsible for periodically sending LFN Time Sync frames in
+> order to maintain broadcast timing in LFN children.
+
+### `0x33 SET_FHSS_ASYNC`
+
+Configure asynchronous transmissions for network discovery.
+
+ - `uint32_t tx_duration_ms`  
+    Maximum number of milliseconds the RCP is allowed to stay in continuous TX
+    for an async transmission. If that duration is exceeded, the async
+    transmission is split into chunks of that duration until all channels have
+    been used. This mechanism makes the RCP radio available for other tasks
+    between the async chunks, which becomes relevant for PHY configurations
+    with many channels. By default, the value is set to `0xffffffff`, which
+    disables this feature.
+
+  - `uint8_t chan_mask_len`  
+     Number of bytes in the following channel mask.
+
+  - `uint8_t chan_mask[]`  
+     Bitmask of channels to use for transmission. See the same field in
+     ["Channel Sequence"][chan-seq] for more details.
+
+## Security
+
+The RCP is responsible for encrypting and decrypting IEEE 802.15.4 frames.
+MAC layer data appears in cleartext in HIF commands so integrators are expected
+to secure the serial link, for example using [CPC with link encpryption][cpc-sec].
```

Comment:
I think we can insist even more here. We could clarify that the system will be exposed to packet injection and eavesdropping if it's not done.